.vscode/settings.json

```diff
@@ -1,3 +1,8 @@
 {
-    "files.insertFinalNewline": true
+    "files.insertFinalNewline": true,
+    "[python]": {
+        "editor.defaultFormatter": "ms-python.autopep8"
+    },
+    "python.formatting.provider": "none",
+    "python.formatting.autopep8Args": ["--ignore", "E402"]
```

Comment:
I'm not a big fan of having a git-tracked file in gitignore.

This might be a reasonable use case for that pattern. Are those settings you added good and reasonable for all vscode users on this project?

Reply:
I mean't to remove this as a git tracked file. these were settings I had in my editor but they could be useful to others (especially the autopep8 override so that it doesn't try to re-arrange imports)